### ISSUE TITLE
[Feature Request] Suggestions for Adding Other Programming Languages

### ISSUE THREAD

**Issue by LixvYang** [vuepress-theme-hope/vuepress-theme-hope#3476]
### Checklist

- [X] I searched existing issues and no one else requests similar feature.
- [X] I think that 25%+ users are positive towards this feature.

### Describe the feature

My blog is mainly about writing code related to go, rust, c++ language code, I would like to add some run buttons to the code area on my blog to run my code for these non-front-end programs, I think it can be achieved by way of webassembly or through online, does the author have any plans to develop a plugin like this at the moment?

**Comment by Mister-Hope** [comment]
No, unless you provide a detailed runtime package which can be easily imported and run in browser

**Comment by LixvYang** [comment]
I think I can release an npm package, the function of npm package is to read the golang program and output the corresponding result, can you embed it for me?

**Comment by Zihan-Hu** [comment]
+1 for C++

**Comment by LixvYang** [comment]
One option I've seen is that each time you run the code is to send an rpc request to an interface provided by your own server, and then the server returns the result, e.g. https://go.dev. @Mister-Hope

**Comment by Mister-Hope** [comment]
Any progress on this?

**Comment by Mister-Hope** [comment]
I just implement kotlin playground with `kotlin-playground`, and I am happy if you can point out other official playground / interpreter packages.
